client/blocks/app-banner/style.scss

```diff
@@ -0,0 +1,68 @@
+.card.app-banner {
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	background-color: $blue-light;
+	color: $blue-dark;
+	/* Needed for banner to appear in front of notifications panel. */
+	z-index: 180;
```

Comment:
Don't forget to move the 180 value to [_z-index.scss](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#z-index).

Let me know if you had any questions.